package.js

```diff
@@ -32,19 +32,25 @@ Package.onTest(function(api) {
   api.use('meteorhacks:fast-render');
   api.use('meteorhacks:inject-data');
   api.use('tmeasday:html5-history-api');
-  api.use('smithy:describe@1.0.0');
+  api.use('smithy:describe@1.0.1');
 
   api.addFiles('test/client/_helpers.js', 'client');
   api.addFiles('test/server/_helpers.js', 'server');
 
-  api.addFiles('test/client/loader.spec.js', 'client');
+  api.addFiles('lib/__tests__/loader.js', ['client', 'server']);
```

Comment:
okay. Right now, I'm in the middle of something. Will do a review tonight. Hope you won't mind that :)